test/unit/services/WorkItemProcessingSchedulerSpec.scala

```diff
@@ -20,8 +20,8 @@ import akka.actor.ActorSystem
 import org.mockito.Mockito._
 import org.scalatest.concurrent.Eventually
 import org.scalatest.concurrent.PatienceConfiguration.{Interval, Timeout}
-import org.scalatestplus.mockito.MockitoSugar
 import org.scalatest.time.{Millis, Seconds, Span}
+import org.scalatestplus.mockito.MockitoSugar
```

Comment:
revert import change

Reply:
needs mockitoSugar to mock notificatiuon logger